templates/python/build.sh

```diff
@@ -1 +1,8 @@
-#!/bin/bash
\ No newline at end of file
+#!/bin/bash
+
+python_templates=$(ls -d */ | sed 's/\//\n/g')
+current_directory=`pwd`
+for template in $maven_templates; do
```

Comment:
should be `python_templates` if we still wanna keep it.

Reply:
Good catch